### ISSUE TITLE
certifiedBy and certifierCredentials not being identified

### ISSUE THREAD

**Issue by clapierre** [daisy/ace#383]
In the latest version of ACE 
Ace (Command line v1.2.7)
Ace (App v1.1.5)

the following metadata as defined in the Accessibility 1.1 specification (which allows for refines pointing to an id) to be included is not being picked up by the ACE checker.

IE:

Missing accessibility Metadata:
a11y:certifiedBy
a11y:certifierCredential

Although inside the EPUB's OPF file the following metadata is provided.

```
    <meta property="dcterms:conformsTo" id="conf">EPUB Accessibility 1.1 - WCAG 2.1 Level AA</meta>
    <meta property="a11y:certifiedBy" id="certifier" refines="#conf">Benetech</meta>
    <meta property="a11y:certifierCredential" refines="#certifier">GCA</meta>

```

**Comment by clapierre** [comment]
@danielweck any way this can be addressed so I can recommend the new metadata for the Accessibility 1.1 Specification to all our GCA partners?

**Comment by danielweck** [comment]
Hello Charles, I am working in a fix for metadata refines. Regards, Daniel

**Comment by clapierre** [comment]
Hi @danielweck any progress on these various refines for metadata?  Would like to be able to start having publishers ready to use the updated metadata once the 1.1 specification becomes a WC3 recommendation which we are now currently voting on in the W3C.

**Comment by danielweck** [comment]
Hello Charles, I pushed a bugfix to enable support for refined accessibility metadata.
I also updated code in both Ace (command line) and Ace App (graphical user interface) to handle `dcterms:conformTo` as per both accessibility specification versions 1.0 and 1.1 (i.e. legacy IDPF WCAG URLs vs. new string labels).

Some background information: the original reason for the wholesale exclusion of `refines` associations was that this is used (for example) to declare the media overlays duration of individual resources instead of the publication as a whole, or (for example) to declare that a `dc:title` is in fact a `title-type` = `subtitle`. The current metadata parsing model discards "refine" information, which is not ideal as this causes a loss of semantics, but this approach has historically greatly simplified processing (notably in the Ace GUI metadata editor).

**Comment by clapierre** [comment]
Thanks @danielweck When will there be a new version of the Ace App? Still shows 1.1.5

**Comment by danielweck** [comment]
Hello Charles, the release is currently blocked by internal updates of a core software component which are causing regressions, I am looking into the technicalities of releasing an intermediary package that would only contain the EPUB a11y specification updates.
